cmd/sriov/main.go

```diff
@@ -202,6 +235,9 @@ func cmdDel(args *skel.CmdArgs) error {
 		// Return nil when LoadConfFromCache fails since the rest
 		// of cmdDel() code relies on netconf as input argument
 		// and there is no meaning to continue.
+		logging.Info("Cannot load config file from cache",
```

Comment:
shouldn't it be error level?

Reply:
Because I originally did not want to print any extra information by default with the introduction of the logging mechanism.  Let me change this though